airmozilla/manage/tests/views/test_picturegallery.py

```diff
@@ -242,6 +242,16 @@ def test_picture_add(self):
         assert picture.size
         eq_(picture.modified_user, self.user)
 
+    def test_picture_add_with_event(self):
+        event = Event.objects.get(title='Test event')
+        url = reverse('manage:event_edit', kwargs={'id': event.id})
+        response = self.client.get(url, {'event': event.id})
+        eq_(response.status_code, 200)
+        ok_(event.title in response.content)
+
+        url = reverse('manage:picturegallery') + '?event=%d' % event.id
+        ok_(url in response.content)
+
```

Comment:
So this isn't testing much. I think what you want to test is **going** to the picture add page, but with the `event` parameter on. E.g. 
```python
# add a picture with an event id on it
url = reverse('manage:picture_add', {'event': event.id)
ok_(event.title in response.content)
ok_(event_edit_url in response.content)
```